src/main/java/Duke.java

```diff
@@ -1,10 +1,19 @@
+import file.FileManager;
+import handler.InputHandler;
+import task.TaskList;
+import ui.UI;
+
+import java.io.FileNotFoundException;
+
 public class Duke {
+    public static void startProgram() {
+            UI.welcomeUser();
+            FileManager.startReading();
+            InputHandler.handleInput();
```

Comment:
You should set the indentation to 4 spaces to ensure consistency and in adherance to the coding standard.